src/models/User.js

```diff
@@ -32,16 +32,6 @@ module.exports = (sequelize, DataTypes) => {
       country: { type: DataTypes.STRING, allowNull: true },
       zip: { type: DataTypes.STRING, allowNull: true },
       phoneNumber: { type: DataTypes.STRING, allowNull: true },
-      supplierId: {
```

Comment:
Why was this user field deleted? by doing so, you break the supplier registration feature. can you provide some clarity as to why it was deleted?

Reply:
Thank you for calling this to my attention Kingsley. I removed it initially so i wouldn't have to point to a supplier table in the earlier integration. It did not call this change to my attention during rebasing, I definitely wouldn't have removed it actively. I've pushed a fix.

Reply:
Great @dchima